app/views/hmis/show.haml

```diff
@@ -1,36 +1,73 @@
 - title = "HMIS #{@type} Source Data"
 - content_for :title, title
 %h1= content_for :title
+- if @imported
+  .well
+    %p
+      Details are shown for imported data.  From right to left, the columns show the results of the import process.  On the right,
```

Comment:
Drop the 'right to left', and order the descriptions to match the table header.

Reply:
See if you like it better now.